### PR TITLE
Movable inventory objects

### PR DESCRIPTION
**Pull request by vladikganja**

### CODE REVIEW

File: creature/player/player.h [hunk 1]
```diff
@@ -10,7 +10,7 @@ class Player : public Creature {
     void action(sf::Event& event, float time, Field* game_field,
                 const std::vector<std::shared_ptr<Creature>>& drawable_creatures);
 
-    Inventory inventory;
+    static inline Inventory inventory;
     float get_satiety() {
```
Comment (akruglov18) [inline creature/player/player.h:14]: Why this field is static?

Comment (vladikganja) [reply creature/player/player.h:14]: we can discuss it in diskord, babe

Comment (akruglov18) [reply creature/player/player.h:14]: Do you DEEPly convinced it that?

File: interfaces/screen/graphic_inventory.h [hunk 2]
```diff
@@ -1,12 +1,14 @@
 #pragma once
 #include "screen.h"
 
+#define NONE_CHOSEN 9223372036854775807
+
```
Comment (akruglov18) [inline interfaces/screen/graphic_inventory.h:5]: C++ provides `std::numeric_limits<T>::max()` and some other functions

File: creature/player/player.h (hunk 1 above)
Comment (vladikganja) [reply creature/player/player.h:14]: unfortunately no;( there are some problems, I'd like to talk about them

File: equipment/items/coins.cpp [hunk 3]
```diff
@@ -16,7 +16,7 @@ void Coins::changeValue(int added) {
 }
 
 void Coins::chooseTexture() {
-    std::vector<int> bounds = {1, 100, 500, 1000, 5000};
+    std::vector<std::size_t> bounds = {1, 100, 500, 1000, 5000};
     auto size = static_cast<int>(bounds.size());
     for (int i = size - 1; i >= 0; i--) {
```
Comment (akruglov18) [inline equipment/items/coins.cpp:21]: `constexpr`, если получится

**Review by akruglov18 — APPROVED** [review]

**Pull request merged**